src/debug.cc

```diff
@@ -73,6 +74,29 @@ HPP_UTIL_LOCAL void makeDirectory(const std::string& filename) {
 
   boost::filesystem::create_directories(dirname);
 }
+
+struct SetVerbosityLevelFromEnvVar {
+  SetVerbosityLevelFromEnvVar() {
+    const char* levelStr = getenv(ENV_LOGGINGLEVEL);
+    if (levelStr) {
+      try {
+        int level = std::stoi(levelStr);
+        if (level >= 0) {
+          setVerbosityLevel(level);
+        } else {
+          std::cerr << ENV_LOGGINGLEVEL << " env var should be positive."
```

Comment:
In english, "positive" means ">0". You probably mean "env var should not be negative".

Reply:
I fixed the error message.